Documentation/Evolution/StronglyTypedCaptures.md

```diff
@@ -33,26 +41,25 @@ if let match = "abcddddefgh".firstMatch(of: regex) {
 }
 ```
 
->***Note:** The `Regex` type is only generic over its captures, but
-`firstMatch(of:)` also includes the entire match at position `.0` in the
-result.*
+>_**Note:** The `Regex` type includes, and `firstMatch(of:)` returns, the entire
+match as the "0th capture"._
```

Comment:
In the rest of the document, by "captures" we refer to second tuple element and onwards. For instance:
```swift
(EntireMatch, Capture...)
```
So it's probably better not to refer to the entire match as the "0th capture".